src/style.css

```diff
@@ -1 +1,61 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
```

Comment:
You are using a `*` selector, which is not recommended due to its impact on performance. Please consider using more specific selectors.